src/ducks/apps/components/Partnership.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import { translate } from 'cozy-ui/react/I18n'
+import { getTranslatedManifestProperty } from 'lib/helpers'
+import ReactMarkdownWrapper from 'ducks/components/ReactMarkdownWrapper'
+
+export class Partnership extends Component {
+  handleBrokenImage = e => {
+    const imgNode = e.nativeEvent.target
```

Comment:
I'm surprised that the React event does not have any target ? We cannot check do `const imgNode = e.target` ?

Reply:
We can't use directly the target property here. React use `SyntheticEvent` which is a layer over the event. Either you persist using `e.persist()` and then use the target or you use the `nativeEvent` property like here:

> [If you find that you need the underlying browser event for some reason, simply use the nativeEvent attribute to get it.](https://reactjs.org/docs/events.html#image-events)